pkgs/applications/editors/emacs/elisp-packages/README.md

```diff
@@ -0,0 +1,116 @@
+# Emacs Lisp Packages Framework
+
+This directory includes a framework for packaging Emacs Lisp (henceforth, Elisp)
+packages to Nixpkgs.
+
+## Bulk Updating
+
+There are basically two methods for updating the Elisp package sets.
+
+### From `nix-community` Overlay
+
+The easiest one is to download and commit the package sets from [`nix-community`
+Emacs Overlay](https://github.com/nix-community/emacs-overlay). The script
+`./update-from-overlay` does all the work:
+
+- It downloads and overwrites the files `elpa-generated.nix`,
+  `elpa-devel-generated.nix`, `recipes-archive-melpa.json`,
+  `nongnu-generated.nix` and `nongnu-devel-generated.nix`;
+- Tests each package set;
+- Commits the changes.
+
+### From local update scripts
```

Comment:
I squashed the scripts, given that they are almost identical.

Reply:
Since some `update-*` scripts are identical, I squashed them in `update-package-sets`.
It would be cool if I could merge `update-melpa` too, but it is slightly different.